frontend/components/FieldAsset/FieldAsset.css

```diff
@@ -0,0 +1,28 @@
+.container {
+  width: 100%;
+  margin-top: 9px;
+  height: 40px;
+  line-height: 25px;
+  font-size: 15px;
+  box-sizing: border-box;
+  border: 1px solid #e4e4e4;
+  border-radius: 5px;
+  outline: 0;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.container[preview] {
```

Comment:
`multiple` is a standard HTML attribute that identifies `<select>` elements with multiple options. `preview` is something we made up.